en/tutorial-base.md

```diff
@@ -536,15 +603,20 @@ class SignupController extends Controller
     }
 }
 ```
+<br/>
 
-We then instantiate the Users class, which corresponds to a User record. The class public properties map to the fields of the record in the users table. Setting the relevant values in the new record and calling `save()` will store the data in the database for that record. The `save()` method returns a boolean value which indicates whether the storing of the data was successful or not.
+At the beginning of the **registerAction** we create an empty user object from the Users class, which manages a User's record. The class's public properties map to the fields of the users table in our database. Setting the relevant values in the new record and calling `save()` will store the data in the database for that record. The `save()` method returns a boolean value which indicates whether the storing of the data was successful or not.
```

Comment:
What I am doing is converting anything that would be quoted " " for visual call out to bold. Things that are code I still use backticks. This might be a bit confusing? I don't want to overuse either one.

The idea is that like in a textbook topic matter specific to a paragraph is usually visually identified with emphasis while other items may be quoted that are supporting but not the thesis.

What are your thoughts?